src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-page-form/index.js

```diff
@@ -126,5 +129,14 @@ export default {
 
             return isSectionDisplay || isBlockDisplay;
         },
+
+        getIsOverridden(slot) {
+            const result = !isEqual(slot.originalConfig, slot.config);
```

Comment:
Similar to my other comment, this check is always true for certain elements, like the `sw-text-editor`. The button correctly disappears after reset changes, however after saving and reloading, it is visible again.